ticker/src/main/java/com/robinhood/ticker/TickerColumnManager.java

```diff
@@ -153,10 +155,16 @@ float getCurrentWidth() {
      * accordingly for the draw procedures.
      */
     void draw(Canvas canvas, Paint textPaint) {
+        TickerColumn prevColumn = null;
```

Comment:
Do you have to save this or can you just do `i > 0 && tickerColumn.get(i - 1).isChanging()`?